### DISCUSSION TITLE
in remotion+nextjs13 `remotion.config.ts` not working?

### DISCUSSION THREAD

**Discussion by winner106** [remotion-dev/remotion#2431] [Q&A] [answered]
according to the doc [https://www.remotion.dev/docs/config](https://www.remotion.dev/docs/config),

> Configuration file
> To configure Remotion, create a `remotion.config.ts` file in the root of your Remotion project.

I created a `remotion.config.ts` file in the root of Remotion project, but it's not working~

**Comment by JonnyBurger** [comment]
Next sentence:

> These options will apply to CLI commands such as npm start and npm run build.

What we mean by this is only to `npx remotion` commands, we will fix the docs!

**Reply by winner106** [reply]
Thanks, server side preview, can I have some way to override webpack config?

**Reply by JonnyBurger** [reply, marked as answer]
@winner106 for this, you ned a second Webpack override, for Next.js: https://nextjs.org/docs/app/api-reference/next-config-js/webpack
